# Behavior subsystem requested command state 
string commandState 
string param1
string param2

